dev_tools/nyaml2nxdl/nyaml2nxdl_backward_tools.py

```diff
@@ -332,6 +333,53 @@ def clean_and_organise_text(self, text, depth):
 
         return text
 
+    def check_and_handle_doc_xref_and_other_doc(self, text, indent):
+        """Check for xref doc which comes as a block of text.
+
+        The doc part bellow is the example how xref comes:
+        '''
+        This concept is related to term `<term>`_ of the <spec> standard.
+        .. _<term>: <url>
+        '''
+        converter as
+        '''
+        <indent>  "xref:
+        <indent>    xpec: <value>
+        <indent>    erm: <value>
+        <indent>    url: <value>"
+        '''
+
+        Parameters
+        ----------
+        text: str
+            plain text.
+        Returns
+        -------
+        str
+            return part of doc as formatted
+        """
+
+        spec, term, url = (None, None, None)
+        xref_key, spec_key, term_key, url_key = ("xref", "spec", "term", "url")
+        before_term = "This concept is related to term `"
+        after_term = "`_ of the"
+        after_spec = " standard."
```

Comment:
I would recommend using a regex for this (there is the unlike case that this does not follow the xref concept but still uses these words).

Reply:
That is a good idea. Thank you for that. I used regex in that part.